### PR TITLE
Adding the ability to skip files

### PR DESCRIPTION
**Pull request by Keboo**
Thank you for this great project!

For my project I have several resx files that I would prefer to not leverage the source generator for. I have added a new option that can be put on the EmbeddedResource that enables ignoring the file completely (since removing the resx as an embedded resource is not an option for me). This is somewhat related to the problem identified in issue #2, but with a more generalized solution.

Happy to make any needed adjustments to this PR

### CODE REVIEW

**Comment by ycanardeau** [comment]
Thank you for this great PR! Looks good to me, but could you please add a section for the new `SkipFile` option in README? You can do it in another PR if you want.

File: Aigamo.ResXGenerator.Tests/SettingsTests.cs [hunk 1]
```diff
@@ -273,7 +274,6 @@ public void FileSettings_RespectsGlobalDefaults()
 		fileOptions.StaticMembers.Should().Be(false);
 		fileOptions.PublicClass.Should().Be(true);
 		fileOptions.PartialClass.Should().Be(true);
-		fileOptions.IsValid.Should().Be(true);
```
Comment (ycanardeau) [inline Aigamo.ResXGenerator.Tests/SettingsTests.cs:276]: Just out of curiosity, why did you remove this assertion?

Comment (Keboo) [reply Aigamo.ResXGenerator.Tests/SettingsTests.cs:276]: Ah, I should have left a comment on it. It is because it is duplicating the last assertion in this test.

**Comment by Keboo** [comment]
> Thank you for this great PR! Looks good to me, but could you please add a section for the new `SkipFile` option to README? You can do it in another PR if you want.

Sure, I can add that.

**Comment by ycanardeau** [comment]
Thanks for the PR!
